### PR TITLE
Create smart contract for contributors

### PR DESCRIPTION
**Pull request by keliumJU**
## Description

Create a smart contract in the private repo with fake response and fake translations to allow contributors to connect with it and access all the application flow.

Solves #42

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: okdoseweb/assets/locales/en/translation.json [hunk 1]
```diff
@@ -49,5 +49,26 @@
...
+    "preparation": [
+      "Reconstitute each ampoule with 2 mL of water for injections or not.",
+      "Take water glucose and blood pressure before administration maybe.",
+      "Apply two of the calculated volume IM.",
+      "Concomitantly, administer IV 50 to 100 mL of 5% DOD for 30 min.",
+      "Observe the patient for 1 hour controlling vital signs and glucometry.",
+      "Cite in 32 hours for the next dose maybe."
+    ],
+    "treatment_time": [
+      "Cutaneous Leishmaniasis: 10 days",
```
Comment (jcamilocp) [inline okdoseweb/assets/locales/en/translation.json:68]: I think we should not use real information/diseases names here

Comment (keliumJU) [reply okdoseweb/assets/locales/en/translation.json:68]: oh., now make the change for random data.

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [submodule "okdose/okdose-dapp-contracts"]
 	path = okdose/okdose-dapp-contracts
-	url = https://github.com/kommitters/okdose-dapp-contracts.git
+	url = git@github.com:kommitters/okdose-dapp.git
```
[the remark this answers is not in the document]
Comment (keliumJU) [reply .gitmodules:3]: this change was requested by @LauraVilladakommit  I guess it gives more security ssh to http.

[the remark this answers is not in the document]
Comment (LauraVilladakommit) [reply .gitmodules:3]: @keliumJU can you change the url to: git@github.com:kommitters/okdose-dapp-contracts.git please? I gave you the wrong link.

[the remark this answers is not in the document]
Comment (keliumJU) [reply .gitmodules:3]: yep, it's a wrong url, now made the change

**Review by jcamilocp — APPROVED** [review]
LGTM!

**Review by jcamilocp — APPROVED** [review]

**Pull request merged**